schema/geometry_schema.json

```diff
@@ -643,22 +842,79 @@
 						"null"
 					]
 				},
+				"window_definition_type": {
+					"type": "string",
+					"enum": [
+						"Single Window",
+						"Multiple Windows",
+						"Window to Wall Ratio"
```

Comment:
I find it confusing to have "WWR" and "Multiple Windows" as separate options. In my mind, WWR implies multiple windows -- and ideally would be implemented as such. Can't we add a property to the WWR option to accommodate multiple windows?

Reply:
For simulation speed a single banded window is best, it’s also harder to satisfy the constraints for window to wall ratio simultaneously with height and width and integer number of windows, so i see these as separate

Reply:
Well, you can use multipliers on windows in EnergyPlus.

Reply:
Bleh